src/js/actions/ui.js

```diff
@@ -509,6 +518,27 @@ define(function (require, exports) {
     zoomInOut.transfers = [zoom];
 
     /**
+     * Set the global resize reference point.
+     *
+     * @param {string} referencePoint One of "tl", "tm", "tr", "cl", "cm", "cr", "bl", "bm", "br".
```

Comment:
Instead of copy-paste the possible value of referencePoint across the project, you can make use of `@typedef`, for example:

```
@typedef {string} ReferencePointString - One of "tl", "tm", "tr", "cl", "cm", "cr", "bl", "bm", "br".
```

and then replace all param docs of referencePoint to 
```
@param {ReferencePointString} referencePoint
```